docs/definition.rst

```diff
@@ -4,210 +4,224 @@ Execution Environment Definition
 The execution environment (EE) definition file supports multiple versions.
 
   * Version 1: Supported by all ``ansible-builder`` versions.
-  * Version 2: Supported by ``ansible-builder`` versions ``1.2`` and higher.
-
-:ref:`Version 2 <version-2>` adds the capability to optionally use and verify
-signed container images. This feature is only supported with the ``podman``
-container runtime.
+  * Version 2: Supported by ``ansible-builder`` versions ``1.2`` and later.
+  * Version 3: Supported by ``ansible-builder`` versions after ``1.2``.
 
 If the EE file does not specify a version, version 1 will be assumed.
 
-.. _version-1:
+.. note::
+
+    This version of the documentation discusses only the latest format version.
+    For further details on older formats, reference previous versions of the
+    documentation.
 
-Version 1 Format
+Version 3 Format
 ----------------
 
-An example version 1 execution environment definition schema is as follows:
+This version of the EE definition file offers substantially more configurability
+and functionality over previous versions.
+
+Below is an example version 3 EE file:
 
 .. code:: yaml
 
     ---
-    version: 1
+    version: 3
 
     build_arg_defaults:
-      EE_BASE_IMAGE: 'quay.io/ansible/ansible-runner:latest'
-
-    ansible_config: 'ansible.cfg'
+      ANSIBLE_GALAXY_CLI_COLLECTION_OPTS: '--pre'
 
     dependencies:
       galaxy: requirements.yml
-      python: requirements.txt
+      python:
+        - six
+        - psutil
```

Comment:
The `schema_v3` doesn't seem to agree with this documentation.  

```
                "python": {
                    "description": "The python dependency file",
                    "type": "string",
                },
```

Reply:
FWIW, I like what the docs have, as it reduces the dependencies on external files.  But if we did extend the schema to support this, I think we should do it for `galaxy` and `bindep` too.

Reply:
So, this wasn't actually working. Must've broke it when jsonschema support was added. Fixed with tests.